spec/system/task_spec.rb

```diff
@@ -58,23 +55,21 @@
   end
 
   describe 'Task編集' do
+    let!(:task_done) { create(:task, :done) }
+
     context '正常系' do
-      xit 'Taskを編集した場合、一覧画面で編集後の内容が表示されること' do
+      it 'Taskを編集した場合、一覧画面で編集後の内容が表示されること' do
         # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit edit_project_task_path(project, task)
```

Comment:
ここも変えないと`let(:task)`が呼ばれてTaskのテストデータに無駄な2件目が作成されてしまうと思います。
`binding.pry`で実行の前後の`Task.count`をご確認ください。


そもそも`let!(:task_done) { create(:task, :done) }`は「既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと」のテストケースの直前に定義すれば十分かもしれません。
```suggestion
        visit edit_project_task_path(project, task_done)
```

Reply:
7568eec でTaskのテストデータが無駄に作成されないように修正しました。